fri/test.py

```diff
@@ -0,0 +1,13 @@
+import requests
+
+url = "http://127.0.0.1:5000/multiple-files-upload"
+
+payload={}
+files=[
+  ('files[]',('example.py',open('/home/amit/Desktop/concore/flaskApi/example.py','rb'),'application/octet-stream'))

```

Comment:
Pls make it a relative path and get the file from the current directory. No hard-coded locations please. They will simply not work outside your laptop.